source/brailleDisplayDrivers/freedomScientific.py

```diff
@@ -247,6 +247,8 @@ def script_toggleRightWizWheelAction(self,gesture):
 			"kb:control+end" : ("br(freedomScientific):dot4+dot5+dot6+brailleSpaceBar",),
 			"kb:alt" : ("br(freedomScientific):dot1+dot3+dot4+brailleSpaceBar",),
 			"kb:alt+tab" : ("br(freedomScientific):dot2+dot3+dot4+dot5+brailleSpaceBar",),
+			"kb:alt+shift+tab" : ("br(freedomScientific):dot1+dot2+dot5+dot6+brailleSpaceBar",),
+			"kb:win+tab" : ("br(freedomScientific):dot2+dot3+dot4+brailleSpaceBar",),

```

Comment:
This one is wrong and breaks the driver currently. it should be windows, not win.

@Oliver2213: Could you provide a pr that fixes this?